owncloudDomain/src/main/java/com/owncloud/android/domain/user/model/UserQuota.kt

```diff
@@ -0,0 +1,37 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Abel García de Prada
+ * Copyright (C) 2020 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.owncloud.android.domain.user.model
+
+import androidx.annotation.VisibleForTesting
+import kotlin.math.roundToLong
+
+data class UserQuota(
+    val available: Long,
+    val used: Long
+) {
+    @VisibleForTesting
+    fun isLimited() = available > 0
+
+    fun getRelative() = if (isLimited()) {
+        val relativeQuota = (used * 100).toDouble() / getTotal()
+        (relativeQuota * 100).roundToLong() / 100.0
+    } else 0.0
+
+    fun getTotal() = if (isLimited()) available + used else 0
```

Comment:
I would have used braces here but up to you 😋  , too much text together `available + used else`...